src/lib/Rest/v1/swagger.yml

```diff
@@ -3817,6 +3817,47 @@ definitions:
                   options:
                     description: "More ldap specific settings (http://php.net/manual/en/function.ldap-set-option.php)."
                     type: object
+  core.v1.SqlSrvUsersEndpoint:
```

Comment:
You probably missed adding SqlSrvUsersEndpoint to the oneOf list, like you did in the openapi specs.